knit/src/Knit/Tokenizer.hs

```diff
@@ -260,25 +267,24 @@ tokenize = fromMaybe noTokenErr . tokenize'
 tokenize'
   :: (KnownSpine components, AllConstrained (ComponentTokenizer components) components)
   => Text
-  -> Maybe [(SSpan, Token components)]
+  -> Maybe (Skipped, [Located (Token components)])
 tokenize' =
   -- 'parseMaybe' runs a parser, returning the result as 'Just' in case of
   -- success and as 'Nothing' in case of failure. It expects the parser to
   -- consume the entire input up to 'eof' and fails otherwise.
   parseMaybe $
     -- We skip unimportant characters in the beginning of the inupt string
     -- and after each token, covering all space around tokens.
-    --
-    -- We alse annotate each token with its source span.
-    pSkip *> many (withSpan pToken <* pSkip)
+    let located span' t skipped = Located span' skipped t
+    in (,) <$> pSkip <*> many (uncurry located <$> withSpan pToken <*> pSkip)
```

Comment:
Minor: why use `uncurry` instead of matching on the tuple directly in `located`?